### ISSUE TITLE
[HOLD #12603] [Image] Seeing blank modal for avatar, when offline reported by @thesahindia 

### ISSUE THREAD

**Issue by kavimuru** [Expensify/App#12707]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed (1):

1. Disconnect the internet connection
2. Navigate to a chat
3. Click on chat header
4. Click on avatar

## Expected Result: 
You should see loading indicator

## Actual Result: 
There's no loading indicator

## Action Performed (2):
1. Go to URL https://staging.new.expensify.com/
2. Login with any account
3. Go offline
4. Upload a new Avatar to your profile
5. Go online
6. Open any chat and open your own details.
7. Tap on the avatar 

## Expected Result:
The photo loads after uploading a new avatar and clicking on details

## Actual Result:
The photo does not load after uploading a new avatar and clicking on details

## Workaround:
unknown

## Platform:

Where is this issue occurring?

- Web
- Desktop App
- Mobile Web

**Version Number:** 1.2.27-3
**Reproducible in staging?:** y
**Reproducible in production?:** y
**Email or phone of affected tester (no customers):** 
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Notes/Photos/Videos:** 
<img width="960" alt="Screenshot (98)" src="https://user-images.githubusercontent.com/43996225/201702937-05e78f79-a7aa-463b-aaa5-9ce5f5d746be.png">

https://user-images.githubusercontent.com/43996225/201702995-cdcff1ad-f743-469a-9b33-9b2800a68b9b.mov

**Expensify/Expensify Issue URL:** 
**Issue reported by:** @thesahindia 
**Slack conversation:** https://expensify.slack.com/archives/C049HHMV9SM/p1668369779537849

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by Beamanator** [comment]
Placing on hold & adding to tracker: https://github.com/Expensify/App/issues/10894

Ideally this will get fixed once image caching works, but we'll keep this open to investigate

**Comment by trjExpensify** [comment]
Makes sense. I'll switch Conor out for myself as the BZ.

**Comment by trjExpensify** [comment]
Caching on native is done, still held on Web/Desktop before we can retest this one.

**Comment by trjExpensify** [comment]
Still on hold.

**Comment by trjExpensify** [comment]
Still on hold. Looking to explore the route of getting the web/desktop caching PR unblocked by fixing it in our RNW fork while we wait on the upstream.

**Comment by trjExpensify** [comment]
No change since the last update.

**Comment by Beamanator** [comment]
Note: Possible fix for me later:
1. Remove `isAuthTokenRequired` from `DetailsPage` -> `AttachmentModal`
2. In `AttachmentView`, render `ImageView` also if `props.sourceURL` starts with `blob:`

**Comment by Beamanator** [comment]
Note: Added another set of reproduction steps to the OP (from here: https://github.com/Expensify/App/issues/14130) so we can close out the other issue

**Comment by trjExpensify** [comment]
Still held.

**Comment by trjExpensify** [comment]
Still waiting on the web/desktop caching PR, but we're real close to getting that merged from looking at it!

**Comment by trjExpensify** [comment]
App PR for web/desktop image caching is still in review: https://github.com/Expensify/App/pull/13036

**Comment by trjExpensify** [comment]
No change, Melv.

**Comment by trjExpensify** [comment]
Still held on the caching PR which is being held by a CORS issue.

**Comment by trjExpensify** [comment]
Still held on web caching, Melv.

**Comment by trjExpensify** [comment]
No change.

**Comment by trjExpensify** [comment]
Samesies!

**Comment by trjExpensify** [comment]
Same Melv!

**Comment by trjExpensify** [comment]
Same Melv!

**Comment by trjExpensify** [comment]
Still held on web caching, Melv.

**Comment by trjExpensify** [comment]
Still held on web image caching, but we're making progress there with the CORS issue dependency blocking that.

**Comment by trjExpensify** [comment]
Still held, but the web image caching PR is in review.

**Comment by trjExpensify** [comment]
Image caching PR was deployed and reverted. Awaiting this now before re-implementing: https://github.com/Expensify/App/issues/32703

**Comment by trjExpensify** [comment]
Damn, we so unlucky Melv! The image caching PR got reverted again, so this is still held.

**Comment by trjExpensify** [comment]
No change.

**Comment by trjExpensify** [comment]
All good here. Closing! 

https://github.com/Expensify/App/assets/16232057/900991a5-212e-4a0f-a609-3339fe5f3aa1

**Comment by trjExpensify** [comment]
Payment summary: 

- $50 to @thesahindia for reporting the bug. Eligible as prior to the program change. 👍

**Comment by JmillsExpensify** [comment]
$250 approved for @thesahindia
